pkg/agent/qrm-plugins/cpu/dynamicpolicy/policy.go

```diff
@@ -100,22 +101,28 @@ type DynamicPolicy struct {
 
 	// those are parsed from configurations
 	// todo if we want to use dynamic configuration, we'd better not use self-defined conf
-	enableCPUAdvisor              bool
-	reservedCPUs                  machine.CPUSet
-	cpuAdvisorSocketAbsPath       string
-	cpuPluginSocketAbsPath        string
-	extraStateFileAbsPath         string
-	enableCPUIdle                 bool
-	enableSyncingCPUIdle          bool
-	reclaimRelativeRootCgroupPath string
-	qosConfig                     *generic.QoSConfiguration
-	dynamicConfig                 *dynamicconfig.DynamicAgentConfiguration
-	podDebugAnnoKeys              []string
-	podAnnotationKeptKeys         []string
-	podLabelKeptKeys              []string
-	transitionPeriod              time.Duration
-	cpuNUMAHintPreferPolicy       string
-	cpuNUMAHintPreferLowThreshold float64
+	enableCPUAdvisor                          bool
+	reservedCPUs                              machine.CPUSet
+	cpuAdvisorSocketAbsPath                   string
+	cpuPluginSocketAbsPath                    string
+	extraStateFileAbsPath                     string
+	enableReclaimNUMABinding                  bool
+	enableCPUIdle                             bool
+	enableSyncingCPUIdle                      bool
+	reclaimRelativeRootCgroupPath             string
+	numaBindingReclaimRelativeRootCgroupPaths map[int]string
+	qosConfig                                 *generic.QoSConfiguration
+	dynamicConfig                             *dynamicconfig.DynamicAgentConfiguration
+	podDebugAnnoKeys                          []string
+	podAnnotationKeptKeys                     []string
+	podLabelKeptKeys                          []string
+	transitionPeriod                          time.Duration
+	cpuNUMAHintPreferPolicy                   string
+	cpuNUMAHintPreferLowThreshold             float64
+
+	reservedReclaimedCPUsSize                 int
```

Comment:
Why does the implementation of RNB require modifications to the reservedReclaim logic?